nifi-commons/nifi-expression-language/src/test/java/org/apache/nifi/attribute/expression/language/TestQuery.java

```diff
@@ -2079,6 +2079,34 @@ public void testPadRight() {
         verifyEmpty("${nullString:padRight(10, \"@\")}", attributes);
     }
 
+    @Test
+    public void testRepeat() {
+        final Map<String, String> attributes = new HashMap<>();
+        attributes.put("str", "abc");
+
+        verifyEquals("${not_exist:repeat(1, 2)}", attributes, "");
+        verifyEquals("${str:repeat(1, 2)}", attributes, "abc");
```

Comment:
This line is not deterministic, as the potential results are `abc` and `abcabc`. I will fix when I merge.

Reply:
Oops meant to change that when I made the `maxRepeats` inclusive. Good catch, and thanks for the fix (or I can certainly do it if you prefer)!